apps/bfd-pipeline/bfd-pipeline-idr/model.py

```diff
@@ -843,6 +861,78 @@ def _current_fetch_query() -> str:
             {{ORDER_BY}}
         """
 
+class IDRClaimProfessional(IdrBaseModel):
```

Comment:
Nit: we've been formatting it this way so far. I don't really care if it's `IDR` or `Idr` as long as they're consistent though.

```suggestion
class IdrClaimProfessional(IdrBaseModel):
```